api/app/handlers/v1/__init__.py

```diff
@@ -25,9 +25,11 @@
 async def handle_webhook(webhook_data: str) -> AsyncGenerator[Flow, None]:
     plugin_uuid = extract_reference_id(webhook_data)
     if not plugin_uuid:
-        raise ValueError("Plugin UUID not found in webhook data")
+        return
```

Comment:
Add a comment saying that you are ignoring and why

Reply:
Some webhooks send sample requests as a heartbeat. We need to ignore those requests. One way of doing that is ignore the requests with no `JB_IDENTIFIER`.

Reply:
No add this into the code for future reference.